src/Files.App/Helpers/MenuFlyout/ContextFlyoutItemHelper.cs

```diff
@@ -378,42 +379,30 @@ private static bool Check(ContextMenuFlyoutItemViewModel item, CurrentInstanceVi
 					ShowItem = itemsSelected && showOpenItemWith
 				},
 				new ContextMenuFlyoutItemViewModelBuilder(commands.OpenFileLocation).Build(),
-				new ContextMenuFlyoutItemViewModel()
+				new ContextMenuFlyoutItemViewModelBuilder(commands.OpenDirectoryInNewTabAction)
 				{
-					Text = "OpenInNewTab".GetLocalizedResource(),
-					OpacityIcon = new OpacityIconModel()
-					{
-						OpacityIconStyle = "ColorIconOpenInNewTab"
-					},
-					Command = commandsViewModel.OpenDirectoryInNewTabCommand,
-					ShowItem = itemsSelected && selectedItems.Count < 5 && areAllItemsFolders && userSettingsService.GeneralSettingsService.ShowOpenInNewTab,
-					ShowInSearchPage = true,
-					ShowInFtpPage = true,
-					ShowInZipPage = true,
-				},
-				new ContextMenuFlyoutItemViewModel()
+					IsVisible =
```

Comment:
Isn't this supposed to be handled by ContextMenuFlyoutItemViewModelBuilder?

Reply:
Fixed